i18n/en.yaml

```diff
@@ -24,3 +24,12 @@
   translation:
     one: "One post"
     other: "{{ .Count }} posts"
+
+- id: sidebarCategories
+  translation: "Categories"
+
+- id: sidebarFeaturedTags
+  translation: "Featured Tags"
+
+- id: sidebarRecommendedSeries
+  translation: "Recommend Series"
```

Comment:
This looks like a typo too, should be `RecommendedSeries` indeed. Just like the the variable name

Reply:
team work, good catch!